cpp/src/arrow/sparse_tensor.h

```diff
@@ -217,10 +217,19 @@ class ARROW_EXPORT SparseTensor {
 // ----------------------------------------------------------------------
 // SparseTensorImpl class
 
+namespace internal {
+
+Status MakeSparseTensorFromTensor(const Tensor& tensor,
```

Comment:
The CI failure on AppVeyor will be resolved by adding `ARROW_EXPORT` to this.

Reply:
done